### ISSUE TITLE
Installing pyautogui fails

### ISSUE THREAD

**Issue by Kochise** [asweigart/pyautogui#621]
See here https://github.com/pypa/pip/issues/10455

**Comment by asweigart** [comment]
Fixed in pytweening 1.0.4. I used an old boilerplate setup.py for that module that caused install errors sometimes. You should be able to install pyautogui now, which will pull the latest version of pytweening.

**Comment by Kochise** [comment]
Hi, thank for the update, seems to work, indeed :

```batch
Microsoft Windows [version 10.0.19042.1165]
(c) Microsoft Corporation. Tous droits réservés.

C:\Program Files\Spyder\Python>python -m pip install --upgrade pip
Requirement already satisfied: pip in c:\program files\spyder\python\lib\site-packages (21.2.4)

C:\Program Files\Spyder\Python>python -m pip install --upgrade pyautogui
Collecting pyautogui
  Using cached PyAutoGUI-0.9.53.tar.gz (59 kB)
Collecting pymsgbox
  Using cached PyMsgBox-1.0.9.tar.gz (18 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting PyTweening>=1.0.1
  Downloading pytweening-1.0.4.tar.gz (14 kB)
Collecting pyscreeze>=0.1.21
  Downloading PyScreeze-0.1.28.tar.gz (25 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting pygetwindow>=0.0.5
  Downloading PyGetWindow-0.0.9.tar.gz (9.7 kB)
Collecting mouseinfo
  Downloading MouseInfo-0.1.3.tar.gz (10 kB)
Collecting pyrect
  Downloading PyRect-0.1.4.tar.gz (15 kB)
Requirement already satisfied: Pillow>=5.2.0 in c:\program files\spyder\pkgs (from pyscreeze>=0.1.21->pyautogui) (8.3.2)
Collecting pyperclip
  Downloading pyperclip-1.8.2.tar.gz (20 kB)
Building wheels for collected packages: pyautogui, pygetwindow, pyscreeze, PyTweening, mouseinfo, pymsgbox, pyperclip, pyrect
  Building wheel for pyautogui (setup.py) ... done
  Created wheel for pyautogui: filename=PyAutoGUI-0.9.53-py3-none-any.whl size=36613 sha256=699aa24a055beb3ca3958eabc3cc57b3909e18227d32509345bb25466abb812b
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\23\db\81\a14b5eca81ccb97c15e5bbea8d5394b8cbf6b36451d89dd648
  Building wheel for pygetwindow (setup.py) ... done
  Created wheel for pygetwindow: filename=PyGetWindow-0.0.9-py3-none-any.whl size=11080 sha256=d79ca857f131853551194730d6be3bfd30e0c2c640ca170d930e2fec516e10c7
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\91\7e\35\62d9062a06cfc46fea39e26860253da36f258b3f3fd96c91c3
  Building wheel for pyscreeze (PEP 517) ... done
  Created wheel for pyscreeze: filename=PyScreeze-0.1.28-py3-none-any.whl size=13023 sha256=8ed5274b6ae06cac18c256f52661f28075a728d9932bd648f6e33f8dd3fd4def
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\09\85\24\8db624ac125598fc40c4fbbff9d3b42e1c464d168567f72a42
  Building wheel for PyTweening (setup.py) ... done
  Created wheel for PyTweening: filename=pytweening-1.0.4-py3-none-any.whl size=5854 sha256=6f3a3125d5acfce9443dabb5c11bdae89b227657a0ca6e32d730e549c952f0a5
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\13\0b\3b\73efc9b0421547a03ed3208c92a88ccddae2ce853c1e6da7e9
  Building wheel for mouseinfo (setup.py) ... done
  Created wheel for mouseinfo: filename=MouseInfo-0.1.3-py3-none-any.whl size=10906 sha256=54bf1f93b5cd0ff81b758288b312e933b654d5eb5193402aadc4bbdc144a6f68
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\43\9a\7f\373736bf37b94b358be41fb2f317a0f9940a1dcb7a733e1707
  Building wheel for pymsgbox (PEP 517) ... done
  Created wheel for pymsgbox: filename=PyMsgBox-1.0.9-py3-none-any.whl size=7420 sha256=e5e7844f7e6715893d37f1b1179b2c7f8ba4a5be1bf6c63d68ba526dd9f021fd
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\39\eb\0e\1587e06da34adf675ad26a113da25df542991f5095303374ef
  Building wheel for pyperclip (setup.py) ... done
  Created wheel for pyperclip: filename=pyperclip-1.8.2-py3-none-any.whl size=11136 sha256=89a880bbe78077049c871cf4f7465b92eece1da841adb1d74674653a6c38040d
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\9f\18\84\8f69f8b08169c7bae2dde6bd7daf0c19fca8c8e500ee620a28
  Building wheel for pyrect (setup.py) ... done
  Created wheel for pyrect: filename=PyRect-0.1.4-py2.py3-none-any.whl size=9547 sha256=bfc03a5be87ace6f7bb6c37238e0aea34084a08254f03ce7cfe64243b46e2a93
  Stored in directory: c:\users\dkoch\appdata\local\pip\cache\wheels\97\5f\8e\6f26a5b00d46679ee2391a3542334274ce8bdaf7c6b0f3504c
Successfully built pyautogui pygetwindow pyscreeze PyTweening mouseinfo pymsgbox pyperclip pyrect
Installing collected packages: pyrect, pyperclip, PyTweening, pyscreeze, pymsgbox, pygetwindow, mouseinfo, pyautogui
Successfully installed PyTweening-1.0.4 mouseinfo-0.1.3 pyautogui-0.9.53 pygetwindow-0.0.9 pymsgbox-1.0.9 pyperclip-1.8.2 pyrect-0.1.4 pyscreeze-0.1.28

C:\Program Files\Spyder\Python>

```
